disjfa/mozaic-bundle/0.1/post-install.txt

```diff
@@ -0,0 +1,7 @@
+<bg=blue;fg=white>                              </>
+<bg=blue;fg=white> Mozaic bundle configuration  </>
+<bg=blue;fg=white>                              </>
+
+  * Modify your UNSPLASH_APPLICATION_ID and UNSPLASH_APPLICATION_SECRET config in <fg=green>.env</>
```

Comment:
"Modify  **the**..."

"...CATION_SECRET parameters in..."

And do not use colors on ".env" (we are trying to restrict the use of colors)

Reply:
done